design/src/player-actions/intent.md

```diff
@@ -0,0 +1,79 @@
+# Intent
+
+**Intent** represents the hive mind's collective willpower and focus.
+It regenerates over time, and can be spent at a significantly faster rate to influence and temporarily improve allied organisms.
+
+## Using intent
+
+Intent is primarily spent on spell-like **abilities**:
+
+1. Select which ability you would like to use.
+2. Use your cursor to target an area.
+3. Click and hold, spending intent at a fixed rate in that area.
+4. The longer you hold down the mouse button, the more the effect strengthens, or the longer the effect lasts.
+
+Abilities are intended to serve three design roles:
+
+1. Give the players interesting and fun micromanagement tasks to do as they watch their [contraption](../high-level/creative-automation.md) run.
+2. Give players tools to deal with crises.
+3. Provide an escape hatch for when players want or need more direct control.
+
+Small amounts of intent are also spent on other actions that reflect "collective intelligence":
+
+- [zoning](zoning.md), with higher priority zoning costing more intent
+- [terraforming](terraforming.md)
+- when altering the [signal configuration](../signals/configuring-properties.md) of a strain
+- changing the [recipe](../production-chains/recipes.md) of a building
+
+This is both thematically appropriate and adds a gentle price to micromanagment.
+These costs should be very low, and rapidly recovered: only planning or reconfiguring huge swathes of your factory at once should actually require waiting for intent to recover.
+
+[Purely informational hive mind abilities](informational-tools.md) never cost intent (even if thematically it might be defensible) in order to reduce player frustration.
+
+## Abilities
+
+Intent can be used for several abilities, each of which is [unlocked](intent.md#unlocking-and-upgrading) as the player plays the game.
+They are listed here in the approximate unlock order.
+
+Abilties are always **dual**: their effect can always be thematically mirrored.
+The more straightforward positive effect is always on the left mouse button,
+while the negative effect is on the right mouse button.
```

Comment:
We should already think about gamepad controls as well :)

Reply:
Agreed! Plan is to do a full UI mockup + keybindings soon.